web/portal/client/cypress/fixtures/Friend/getCollection.json

```diff
@@ -1,12 +1,14 @@
 {
   "body": [
     {
-      "name": "testFriend",
-      "description": "",
-      "priority": 1,
-      "directConnectivity": "yes",
-      "id": 1,
-      "domain": 3
+        "name": "testFriend",
+        "description": "",
+        "priority": 1,
+        "directConnectivity": "yes",
+        "id": 1,
+        "company": 1,
+        "domain": 3,
+        "interCompany": null
```

Comment:
This is not testing all existing Friend types. I would at least add interpbx friends to this list like API rest fixtures have: https://github.com/irontec/ivozprovider/blob/bleeding/web/rest/client/features/provider/friend/getFriend.feature#L15-L38